modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -127,12 +132,35 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
             $visit_array =Utility::getExistingVisitLabels();
             $visit_array = array_combine($visit_array, $visit_array);
             $this->tpl_data['visitLabels'] = $visit_array;
+            if ($user->hasPermission('access_all_profiles')) {
+                // get the list of study sites - to be replaced by the Site object
+                $list_of_sites = Utility::getSiteList();
+                if (is_array($list_of_sites)) {
+                    $list_of_sites = array('' => 'All Sites') + $list_of_sites;
+                }
+            } else {
+                // allow only to view own site data
+                $site =& Site::singleton($user->getData('CenterID'));
+                if ($site->isStudySite()) {
+                    $list_of_sites = array();
+                    $user_site     =$user->getData('Site');
+                    $list_of_sites[$user->getData('CenterID')] = $user_site;
+                }
+            }
+            $this->tpl_data['siteList'] = $list_of_sites;
+            // Project list, if applicable
+            if ($config->getSetting("useProjects")==="true") {
+                $projectList = Utility::getProjectList();
+                $this->tpl_data['projectList'] = $projectList;
+            }
 
             $incompleteCandidates = $this->
                  getIncompleteCandidates(
                      $instrument,
                      $_REQUEST['visit_label'],
-                     $candID
+                     $candID,
+                     $_REQUEST['site'],
```

Comment:
Before using any values from `$_REQUEST` they should be sanitized for malicious input, especially Javascript.

Since these values are being submitted directly into the page they are vulnerable to a [Reflected XSS attack](https://www.hacksplaining.com/exercises/xss-reflected). 

Going forward, we should prevent against this by [rejecting malformed input](https://github.com/aces/Loris/pull/2775), or, less preferably, sanitizing the output using e.g. [htmlspecialchars()](http://php.net/manual/en/function.htmlspecialchars.php).